### PR TITLE
Added networking docs to dedicated

### PR DESCRIPTION
**Pull request by sheriff-rh**
Preview build: 
http://file.rdu.redhat.com/~antaylor/dedicatednetworking/networking/understanding-networking.html

@wgordon17  please take a look. On the `configuring network policy` page, i left the "Adding network policy objects..." for now until we get confirmation about changes. Everything else has been added as discussed.

### CODE REVIEW

**Comment by wgordon17** [comment]
LGMT 👍🏻

File: _topic_map.yml [hunk 1]
```diff
@@ -310,19 +310,21 @@ Topics:
...
 - Name: Understanding the DNS Operator
   File: dns-operator
+  Distros: openshift-enterprise,openshift-origin
 - Name: Understanding the Ingress Operator
   File: ingress-operator
+  Distros: openshift-enterprise,openshift-origin
 - Name: Configuring network policy
   File: configuring-networkpolicy
-  Distros: openshift-origin,openshift-enterprise
 - Name: Multiple networks
```
Comment (kalexand-rh) [inline _topic_map.yml]: Is this one valid for dedicated?

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 2]
```diff
@@ -26,3 +27,9 @@ cluster and receive network traffic from those Pods and Services.
 * The _subnet_ mode provides a flat Pod network where every Pod can
 communicate with every other Pod and Service. The network policy mode provides
 the same functionality as the subnet mode.
+endif::[]
+ifdef::openshift-dedicated[]
+{product-title} SDN only supports the _network policy_ mode, which allows project
+administrators to configure their own isolation policies using
```
Comment (kalexand-rh) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: s/only support/supports only
s/using/by using

**Review by kalexand-rh — COMMENTED** [review]
Two small picks and a sanity check, then it LGTM.

File: _topic_map.yml (hunk 1 above)
Comment (sheriff-rh) [reply _topic_map.yml]: Yes, that is one we are adding in for dedicated.

File: networking/openshift-sdn/about-openshift-sdn.adoc (hunk 2 above)
Comment (sheriff-rh) [reply networking/openshift-sdn/about-openshift-sdn.adoc]: Fixed!

File: networking/openshift-sdn/about-openshift-sdn.adoc [hunk 3]
```diff
@@ -26,3 +27,9 @@ cluster and receive network traffic from those Pods and Services.
 * The _subnet_ mode provides a flat Pod network where every Pod can
 communicate with every other Pod and Service. The network policy mode provides
 the same functionality as the subnet mode.
+endif::[]
+ifdef::openshift-dedicated[]
+{product-title} SDN supports only the _network policy_ mode, which allows project
```
Comment (jboxman) [inline networking/openshift-sdn/about-openshift-sdn.adoc]: @sheriff-rh, I don't think there's an OKD SDN, so using `{product-title}` token here doesn't make sense. OKD would use the same Pod network provider as OCP.

Comment (wgordon17) [reply networking/openshift-sdn/about-openshift-sdn.adoc]: @jboxman This is OSD (openshift dedicated), which _should_ have a `{product-title}`

Comment (kalexand-rh) [reply networking/openshift-sdn/about-openshift-sdn.adoc]: Eh, it should be "OpenShift SDN." I'll open a PR to fix it.

Comment (kalexand-rh) [reply networking/openshift-sdn/about-openshift-sdn.adoc]: https://github.com/openshift/openshift-docs/pull/18415

**Comment by vikram-redhat** [comment]
@sheriff-rh did this get manually CPed to 4.1 branch?

**Pull request merged**